OOVPADatabase/XOnline/5659.inl

```diff
@@ -45,3 +45,46 @@ OOVPA_XREF(XoUpdateLaunchNewImageInternal, 5659, 11,
 
         { 0x79, 0xE8 },
 OOVPA_END;
+
+// ******************************************************************
+// * CXo::XOnlineMatchSearchGetResults
+// ******************************************************************
+OOVPA_XREF(CXo_XOnlineMatchSearchGetResults, 5659, 12,
```

Comment:
Recommend to use unique signature mentioned in 4831 file and remove this signature.

Reply:
Done.